src/main/java/org/apache/sysds/runtime/matrix/data/LibCommonsMath.java

```diff
@@ -113,9 +134,31 @@ else if (opcode.equals("eigen_qr"))
 			return computeEigenQR(in, threads);
 		else if (opcode.equals("svd"))
 			return computeSvd(in);
+		else if (opcode.equals("fft"))
+			return computeFFT(in);
+		else if (opcode.equals("ifft"))
+			return computeIFFT(in, null);
```

Comment:
you can overload the computeIFFT function to remove this null argument.

Reply:
did this for computeIFFT and computeIFFT_LINEARIZED